scripts/install.sh

```diff
@@ -44,7 +48,8 @@ echo "Create the directory - \"${DIR}\" successfully."
 echo "Downloading the release file - \"${ARTIFACT}\" ..."
 curl -sL ${ARTIFACT_URL} -o ${DIR}/${ARTIFACT}
 echo "Downloaded \"${ARTIFACT}\""
-tar -zxf ${DIR}/${ARTIFACT} -C easegress 
+tar -zxf ${DIR}/${ARTIFACT} -C "${DIR}"
+rm ${DIR}/${ARTIFACT}
```

Comment:
Is there need `-f` option to `rm` command?

Sometimes there will have `alias rm='rm -i'` configuration in system.

Reply:
hi @grootpiano, perhaps it would be safer if we removed this command.

Reply:
updated, PTAL

Reply:
LGTM. However, I'm not Easegress's maintainer, yet.

So, please request for haoel's review for double check. 😄